Code/GraphMol/FileParsers/MaeMolSupplier.cpp

```diff
@@ -0,0 +1,144 @@
+//
+//  Copyright (C) ????  Who to put here when contributing to rdkit?
```

Comment:
To this point we’ve always had the author put themselves in. Except when people copy files and forget to change it of course. :-)
I’m fine with sticking with that plan.

Reply:
Sounds good